crates/pera-core/src/dwallet_mpc/dwallet_mpc_service.rs

```diff
@@ -12,7 +12,7 @@ use tokio::sync::{watch, Notify};
 use tracing::error;
 use typed_store::Map;
 
-const READ_INTERVAL_SECS: u64 = 5;
+const READ_INTERVAL_MILLIS: u64 = 100;
```

Comment:
```suggestion
const READ_INTERVAL_MS: u64 = 100;
```
more common this way.